components/datetime/src/options/components.rs

```diff
@@ -489,6 +492,183 @@ impl From<TimeZoneName> for Field {
+                                //     'ccc',
+                                //     'ccc d. MMM',
+                                //     'ccc d. MMMM',
+                                //     'cccc d. MMMM y',
+                                //     'd, ccc',
+                                //     'cccနေ့',
+                                //     'ccc, d MMM',
+                                //     "ccc, d 'de' MMMM",
+                                //     "ccc, d 'de' MMMM 'de' y",
+                                //     'ccc, h:mm B',
+                                //     'ccc, h:mm:ss B',
+                                //     'ccc, d',
+                                //     "ccc, dd.MM.y 'г'.",
+                                //     'ccc, d.MM.y',
+                                //     'ccc, MMM d. y'
+                                unimplemented!("Numeric stand-alone fields are not supported.")
+                            }
+                            FieldLength::Abbreviated => Text::Short,
+                            FieldLength::Wide => Text::Long,
+                            FieldLength::Narrow | FieldLength::Six => Text::Narrow,
+                        },
+                        fields::Weekday::Local => unimplemented!("fields::Weekday::Local"),
+                    });
+                }
+                FieldSymbol::DayPeriod(_) => {
+                    // Day period does not affect the resolved components.
+                }
+                FieldSymbol::Hour(hour) => {
+                    bag.hour = Some(match field.length {
+                        FieldLength::TwoDigit => Numeric::TwoDigit,
+                        _ => Numeric::Numeric,
+                    });
+                    bag.preferences = Some(preferences::Bag {
+                        hour_cycle: Some(match hour {
+                            fields::Hour::H11 => preferences::HourCycle::H11,
+                            fields::Hour::H12 => preferences::HourCycle::H12,
+                            fields::Hour::H23 => preferences::HourCycle::H23,
+                            fields::Hour::H24 => preferences::HourCycle::H24,
+                        }),
+                    });
+                }
+                FieldSymbol::Minute => {
+                    bag.minute = Some(match field.length {
+                        FieldLength::TwoDigit => Numeric::TwoDigit,
+                        _ => Numeric::Numeric,
+                    });
+                }
+                FieldSymbol::Second(second) => {
+                    bag.second = Some(match second {
+                        fields::Second::Second => match field.length {
+                            FieldLength::TwoDigit => Numeric::TwoDigit,
+                            _ => Numeric::Numeric,
+                        },
+                        fields::Second::FractionalSecond => {
+                            unimplemented!("fields::Second::FractionalSecond. #1360")
```

Comment:
> praise

Thanks for providing strings to all of the `unimplemented!()` macro calls, and also for adding issue numbers where relevant.